src/main/java/edu/greenblitz/tobyDetermined/subsystems/LED.java

```diff
@@ -31,45 +30,51 @@ public static void init(){
         }
     }
 
-    public void setColor(Color color) {
-        for (int i = 0; i < this.ledBuffer.getLength(); i++) {
-            this.ledBuffer.setLED(i, color);
-            SmartDashboard.putNumber("led num", i);
-        }
-        this.addressableLED.setData(ledBuffer);
-    }
     public void setColor(int i, Color color) {
```

Comment:
IMO `i` is a good name (mostly) for iterators, consider changing this to be more readable